sections/04_study.md

```diff
@@ -709,101 +709,92 @@ the next few years.
+into homology/protein family classification in 2007, and therefore, deep
+learning is deeply rooted in functional classification methods.
+
+One of the first techniques of “de novo” genome binning used self-organizing
+maps, a type of NN [@tag:Abe]. Essinger et al. use ART, a neural network
+algorithm called Adaptive Resonance Theory, to cluster similar genomic fragments
+and showed that it has better performance than K-means. However, other methods
+based on interpolated Markov models [@tag:Salzberg] have performed better than
+these early genome binners. Neural networks can be slow, and therefore, have had
+limited use for reference-based taxonomic classification, with TAC-ELM
+[@tag:TAC-ELM] being the only NN-based algorithm to taxonomically classify
+massive amounts of metagenomic data. Also, neural networks can fail to perform
+if there are not enough training examples, which is the case with taxonomic
+classification (since only ~10% of estimated species have been sequenced). An
+initial study successfully applied neural networks to taxonomic classification
+of 16S rRNA genes, with convolutional networks providing about 10% accuracy
+genus-level improvement over RNNs and even random forests [@tag:Mrzelj].
+However, this study performed 10-fold cross-validation on only 3000 sequences.
+
+Neural network uses for classifying phenotype from microbial composition are
+just beginning. A standard multi-layer perceptron (MLP) was able to classify
+wound severity from microbial species present in the wound
+[@doi:10.1016/j.bjid.2015.08.013]. Recently, Ditzler et al. associated soil
+samples with pH level using MLPs, deep-belief networks, and recursive neural
+networks [@tag:Ditzler3]. Besides classifying the samples appropriately, they
+showed that internal phylogenetic tree nodes inferred by the networks were
+appropriate features representing low/high pH. Thus, hidden nodes might provide
+biological insight as well as new features for future metagenomic sample
+comparison. Also, an initial study has shown promise of these networks for
+diagnosing disease [@tag:Faruqi].
+
+Challenges remain in applying deep neural networks to metagenomics problems.
+They are not yet ideal for phenotype classification because most studies contain
+tens of samples (~20-40) and hundreds or thousands of features (aka species).
+Such underdetermined, or ill-conditioned, problems are still a challenge for
+deep neural networks that require many more training examples than features to
+sufficiently converge the weights on the hidden layers. Also, due to convergence
+issues (slowness and instability due to large neural networks modeling very
+large datasets [@arxiv:1212.0901v2]), taxonomic classification of reads from
+whole genome sequencing seems out of reach at the moment for deep neural
+networks -- due to only thousands of full-sequenced genomes as compared to
+hundreds of thousands of 16S rRNA sequences available for training.
```

Comment:
No, that only thousands of whole genomes are available for training (and have a lot of features), while 16S have hundreds of thousands of training examples (with less features for characterization).